eo-maven-plugin/src/main/java/org/eolang/maven/LintMojo.java

```diff
@@ -89,10 +90,10 @@ public final class LintMojo extends SafeMojo {
     void exec() throws IOException {
         final long start = System.currentTimeMillis();
         final Collection<ForeignTojo> tojos = this.scopedTojos().withShaken();
-        final ConcurrentHashMap<Severity, Integer> counts = new ConcurrentHashMap<>();
-        counts.putIfAbsent(Severity.CRITICAL, 0);
-        counts.putIfAbsent(Severity.ERROR, 0);
-        counts.putIfAbsent(Severity.WARNING, 0);
+        final ConcurrentHashMap<Severity, List<Defect>> counts = new ConcurrentHashMap<>();
+        counts.putIfAbsent(Severity.CRITICAL, new ListOf<>());
+        counts.putIfAbsent(Severity.ERROR, new ListOf<>());
+        counts.putIfAbsent(Severity.WARNING, new ListOf<>());
```

Comment:
@h1alexbel why do we need all this stuff with `List`s? Only retrieving jus one element and take the version? Looks a bit suboptimal to me

Reply:
@maxonfjvipon I agree. But to my knowledge, for now, we can't get the version without defect creation. To find defect, you need to aggregate the _list of defects_, not the count. I also thought about returning version in `Program` from `lints` like this:

```java
final Program program = ...;
final String version = program.version();
```

WDYT? Maybe we should implement such functionality in lints?

Reply:
@h1alexbel yes, it makes sense, let's make a ticket for it

Reply:
@maxonfjvipon created https://github.com/objectionary/lints/issues/324